akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorContextImpl.scala

```diff
@@ -105,7 +105,7 @@ import scala.util.Success
   // context-shared timer needed to allow for nested timer usage
   def timer: TimerSchedulerCrossDslSupport[T] = _timer match {
     case OptionVal.Some(timer) => timer
-    case OptionVal.None =>
+    case _ =>
```

Comment:
that's sad, but ok

Reply:
Yeah, we discussed it somewhere but there is no way for the compiler to know that it is complete (as I remember it, since it is two custom unapply method on some types)